hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -120,11 +120,19 @@ public final class OzoneConfigKeys {
   public static final String OZONE_FS_DATASTREAM_AUTO_THRESHOLD_DEFAULT
       = "4MB";
 
+  /**
+   * Flag to allow hbase-related features and enhancements to be enabled in the first place.
+   */
+  public static final String OZONE_HBASE_ENHANCEMENTS_ENABLED
+      = "ozone.hbase.enhancements.enabled";
+  public static final boolean OZONE_HBASE_ENHANCEMENTS_ENABLED_DEFAULT
+      = false;
+
   /**
    * Flag to enable hsync/hflush.
    */
   public static final String OZONE_FS_HSYNC_ENABLED
-      = "ozone.fs.hsync.enabled";
+      = "ozone.fs.hsync.enabled";  // OzoneConfigKeys is not passed to BOS ?
```

Comment:
Left over comment. Removed. Same as this bullet point.

- `BlockOutputStream` only has `OzoneClientConfig`, which is a filtered version of full config that contains only the client-related configs. And to avoid the ugliness of passing in extra args in the constructor.